third_party/android_deps/BUILD.gn

```diff
@@ -67,5 +67,5 @@ android_aar_prebuilt("com_wireguard_android_java") {
 
 android_aar_prebuilt("com_github_bumptech_disklrucache_java") {
   aar_path = "libs/com_github_bumptech_disklrucache/disklrucache-4.12.0.aar"
-  info_path = "libs/com_github_bumptech_glide_gifdecoder/com_github_bumptech_disklrucache.info"
```

Comment:
is this a mistake?

Reply:
Unrelated to the licensing stuff, but it's fixing an earlier copy/paste mistake: https://bravesoftware.slack.com/archives/C03MMJ4CHEZ/p1679000538767999